### PR TITLE
(no-op) Code commentary and questions

### PR DESCRIPTION
**Pull request by StachuDotNet**

### CODE REVIEW

File: fsharp-backend/src/LibService/Telemetry.fs [hunk 1]
```diff
@@ -315,11 +315,17 @@ let addTelemetry
...
+/// Returns a (random?) executionID used to tie together everything
+/// that happens within a BWD request
+/// </summary>
+/// <remarks>
+/// An execution ID was an int64 ID in the OCaml version, but since we're using
+/// OpenTelemetry from the start, we use the Trace ID instead. This should be used to
+/// create a TraceID for anywhere there's a thread and a trace available. The
+/// execution ID should be constant no matter when this is called in a thread, but for
+/// safety, call it at the top and pass it down.
+/// </remarks>
```
[the remark this answers is not in the document]
Comment (pbiggar) [reply fsharp-backend/src/LibService/Telemetry.fs:328]: Love it! Are `<remarks>` and `<summary>` pre-defined, or is anything allowed?

File: fsharp-backend/src/LibService/Telemetry.fs [hunk 2]
```diff
@@ -315,11 +315,17 @@ let addTelemetry
   |> fun b -> b.SetSampler(sampler)
 
 
-// An execution ID was an int64 ID in the OCaml version, but since we're using
-// OpenTelemetry from the start, we use the Trace ID instead. This should be used to
-// create a TraceID for anywhere there's a thread and a trace available. The
-// execution ID should be constant no matter when this is called in a thread, but for
-// safety, call it at the top and pass it down.
+/// <summary>
+/// Returns a (random?) executionID used to tie together everything
```
Comment (pbiggar) [inline fsharp-backend/src/LibService/Telemetry.fs:319]: It is random in the sense that it is randomly generated. It is not "random" (aka arbitrary) in the sense that it refers to a specific one.

**Review by pbiggar — CHANGES_REQUESTED** [review]
Lovely, thanks! Few small changes and we're good to go.

File: fsharp-backend/src/LibService/Telemetry.fs (hunk 1 above)
[the remark this answers is not in the document]
Comment (StachuDotNet) [reply fsharp-backend/src/LibService/Telemetry.fs:328]: They're pre-defined, though the definition is more of a base/standard than an exhaustive list.

They're referred to as "XML Comments"

In C#/VB you traditionally include at least a `summary` node - other nodes are optional.
In F#, a bare /// comment will be assumed to be a summary node. 

See more information here:
- https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/xml-documentation#recommended-tags

**Comment by StachuDotNet** [comment]
formatting...

**Comment by StachuDotNet** [comment]
@pbiggar ready for merge - will add further commentary in additional PRs

**Comment by pbiggar** [comment]
FYI, there's conflict here.

**Pull request merged**